flang/test/Driver/flang-linker-flags.f90

```diff
@@ -7,6 +7,7 @@
 ! (Linux) specific. The following line will make sure that this test is skipped
 ! on Windows. Ideally we should find a more robust way of testing this.
 ! REQUIRES: shell
+! UNSUPPORTED: x86_64-apple-darwin, x86_64-apple-macos
```

Comment:
This could be simplified as:
```
! UNSUPPORTED: darwin, macos
```
See: https://llvm.org/docs/TestingGuide.html#constraining-test-execution

Reply:
Done. Good point, will make it work for apple silicon as well.